Dockerfile

```diff
@@ -48,9 +35,9 @@ RUN cp config/bioportal_config_env.rb.sample config/bioportal_config_production.
  && cp config/bioportal_config_env.rb.sample config/bioportal_config_test.rb \
  && cp config/database.yml.sample config/database.yml
 
-RUN if [ "${RAILS_ENV}" != "development" ]; then \
-  bundle exec bootsnap precompile --gemfile app/ lib/ && \
-  SECRET_KEY_BASE_DUMMY=1 ./bin/rails assets:precompile; fi
+# RUN if [ "${RAILS_ENV}" != "development" ]; then \
```

Comment:
why did you remove this ? is need for the ontoportal docker project to work.

Reply:
i remove it because it's given an error when building the docker image, like this one here https://github.com/ontoportal-lirmm/bioportal_web_ui/actions/runs/13428532527/job/37515832095#step:8:2342

it give an error because there is no newrelic secret key, maybe i should remove only the last line that contains `SECRET_KEY_BASE_DUMMY` ?

Reply:
the SECRET_KEY_BASE_DUMMY is also important, but newrelic is not important, so that one should be removed while building the image.